resources/store/sessionStore.ts

```diff
@@ -0,0 +1,19 @@
+import create from 'zustand';
+
+export interface SessionInterface {
+    token: string | null,
+    setToken: (token: string) => void,
+    logout: () => void,
+}
+
+export const sessionStore = create<SessionInterface>((set) => ({
+    token: localStorage.getItem("token"),
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/bluespada/go-telescope/pullRequest?prid=9653145)
```suggestion
  token: localStorage.getItem("token"),
```